app/models/procedures_filter.rb

```diff
@@ -28,6 +28,10 @@ def statuses
     params[:statuses].compact_blank if params[:statuses].present?
   end
 
+  def from_publication_date
+    Date.parse(params[:from_publication_date]) if params[:from_publication_date].present?
```

Comment:
d'expérience on va avoir des formats pourris renvoyés, et le parse va péter. En général je fais un rescue nil pour ignorer et faire comme si c'était pas valide. 

Par exemple:


```suggestion
  def from_publication_date
    return if params[:from_publication_date].blank?

    Date.parse(params[:from_publication_date])
  rescue Date::Error
    nil
```

ou même en ligne mais rubocop mais c'est moins visibile: `Date.parse(params[:from_publication_date]) rescue nil`

Reply:
D'accord avec ta proposition. C'est plus prudent :)
Suggestion intégrée dans le commit rebasé : https://github.com/demarches-simplifiees/demarches-simplifiees.fr/pull/8020/commits/a5fa21ca1b7ed93a84fa63361a35e6645b3e7fc4